ios/FluentUI/Core/Theme/FluentTheme.swift

```diff
@@ -21,22 +31,24 @@ public class FluentTheme: NSObject, ObservableObject {
     ///   - typographyOverrides: A `Dictionary` of override values mapped to `TypographyTokens`.
     ///
     /// - Returns: An initialized `FluentTheme` instance, with optional overrides.
-    public init(colorOverrides: [ColorToken: DynamicColor]? = nil,
+    public init(colorOverrides: [ColorToken: UIColor]? = nil,
                 shadowOverrides: [ShadowToken: ShadowInfo]? = nil,
-                typographyOverrides: [TypographyToken: FontInfo]? = nil) {
-        let fixedColorOverrides = colorOverrides?.map({ (key: ColorToken, value: DynamicColor) in
+                typographyOverrides: [TypographyToken: UIFont]? = nil) {
+        let fixedColorOverrides = colorOverrides?.map({ (key: ColorToken, value: UIColor) in
             let newKey = AliasTokens.ColorsTokens(rawValue: key.rawValue)!
-            return (newKey, value)
+            let newValue = value.dynamicColor ?? .init(light: .clear)
```

Comment:
This is a new side-effect/fallback behavior - do we know if this is just addressing a possible nil reference (if I'm reading it right), or does this affect behavior that customers might have taken a dependency on? Probably the former, but wanted to be deliberate about changes in undocumented behavior.

Reply:
I think it actually makes sense to force-unwrap here - good catch